crates/telio-firewall/src/firewall.rs

```diff
@@ -1653,22 +1768,22 @@ pub mod tests {
             };
 
             assert_eq!(fw.tcp.lock().unwrap().get(&conn_key), Some(&TcpConnectionInfo{
-                tx_alive: true, rx_alive: true, conn_remote_initiated: false
+                tx_alive: true, rx_alive: true, conn_remote_initiated: false, remote_seq: 0
             }));
 
             assert_eq!(fw.process_inbound_packet(&make_peer(), &make_tcp(them, us, TcpFlags::FIN)), true);
             assert_eq!(fw.tcp.lock().unwrap().len(), 1);
 
             assert_eq!(fw.tcp.lock().unwrap().get(&conn_key), Some(&TcpConnectionInfo{
-                tx_alive: true, rx_alive: false, conn_remote_initiated: false
+                tx_alive: true, rx_alive: false, conn_remote_initiated: false, remote_seq: 0
             }));
 
             assert_eq!(fw.process_outbound_packet(&make_peer(), &make_tcp(us, them, TcpFlags::FIN)), true);
             assert_eq!(fw.tcp.lock().unwrap().len(), 1);
 
             // update tcp cache entry timeout
             assert_eq!(fw.tcp.lock().unwrap().get(&conn_key), Some(&TcpConnectionInfo{
-                tx_alive: false, rx_alive: false, conn_remote_initiated: false
+                tx_alive: false, rx_alive: false, conn_remote_initiated: false, remote_seq: 0
             }));
 
             // process inbound packet (should not update ttl, because not ACK, but entry should still exist)
```

Comment:
Could you add some unit tests covering `reset_connections()` function? It is much easier to cover various cases in FW via unit-tests then integration.

Reply:
Unit test added